### ISSUE TITLE
No rendering in a safeframe

### ISSUE THREAD

**Issue by levjad** [prebid/Prebid.js#1426]
## Type of issue
Question / Bug

## Description
I'm using the creative code which is provided by prebid.js (integrationExamples/gpt/x-domain/creative.html). Although renderAd is executed several times (or kind of endlessly) no event contains the requested ad(Object). So it's always running in an error within the try/catch block with never rendering anything. Happening with the rubicon adapter and a custom built one.

## Steps to reproduce
Using the creative code as mentioned above for a booking with "Serve into a Safeframe" enabled in the creative (DFP / GPT).

### Expected results
Rendering the ad.

### Actual results
Never rendering something.

## Platform details
Using DFP as a ad server.

**Comment by mkendall07** [comment]
please include a example page. Are you seeing valid bids returned? If no bid is returned then you would see a blank.

**Comment by levjad** [comment]
Unfortunately I can't provide a proper example page because the setup we're using is a bit complex and therefore it's not easy to provide something like that.  I tried to create a simple example but I ended up with everything working properly. So I guess the problem lies within the setup we're using.  

The bids are valid and there definitely an ad. At the end I don't really know why there isn't a "Prebid Response". Maybe I find a solution someday.
